### PR TITLE
[LDAP] Add docs for applyOperations method

### PR DESCRIPTION
**Pull request by mablae**
See PR https://github.com/symfony/symfony/pull/27069

### CODE REVIEW

File: components/ldap.rst [hunk 1]
```diff
@@ -137,6 +137,35 @@ delete existing ones::
     // Removing an existing entry
     $entryManager->remove(new Entry('cn=Test User,dc=symfony,dc=com'));
 
+
+Batch Updating
+______________
+
+Use the EntryManager's ``applyOperations()`` method to update multiple attributes at once::
+
+
```
Comment (xabbuh) [inline components/ldap.rst]: one blank line is enough :)

File: components/ldap.rst [hunk 2]
```diff
@@ -137,6 +137,35 @@ delete existing ones::
     // Removing an existing entry
     $entryManager->remove(new Entry('cn=Test User,dc=symfony,dc=com'));
 
+
+Batch Updating
+______________
+
+Use the EntryManager's ``applyOperations()`` method to update multiple attributes at once::
```
Comment (xabbuh) [inline components/ldap.rst]: [...] entry manager's [...]

Comment (xabbuh) [reply components/ldap.rst]: and we could use the `method` role here:

```rst
:method:`Symfony\\Component\\Ldap\\Adapter\\ExtLdap\\EntryManager::applyOperations`
```

File: components/ldap.rst [hunk 3]
```diff
@@ -137,6 +137,35 @@ delete existing ones::
...
+
+    $entryManager = $ldap->getEntryManager();
+
+    // Adding multiple email adresses at once
+    $entryManager->applyOperations($entry->getDn(), array(
+        new UpdateOpteration(LDAP_MODIFY_BATCH_ADD, 'mail', 'new1@example.com'),
+        new UpdateOpteration(LDAP_MODIFY_BATCH_ADD, 'mail', 'new2@example.com'),
+    ));
+
+Possible operation types are ``LDAP_MODIFY_BATCH_ADD``, ``LDAP_MODIFY_BATCH_REMOVE``, ``LDAP_MODIFY_BATCH_REMOVE_ALL``, ``LDAP_MODIFY_BATCH_REPLACE``. Parameter ``$values`` must be ``NULL`` when using ``LDAP_MODIFY_BATCH_REMOVE_ALL`` operation type.
```
Comment (xabbuh) [inline components/ldap.rst]: we usually split lines after the first word that crosses the 72nd character

File: components/ldap.rst [hunk 4]
```diff
@@ -137,6 +137,35 @@ delete existing ones::
...
+
+.. versionadded:: 4.2
+    The ``applyOperations()`` was introduced in Symfony 4.2.
```
Comment (xabbuh) [inline components/ldap.rst]: The `applyOperations()` method [...]

**Comment by mablae** [comment]
Thanks @xabbuh . I just applied your fixes and rebased it.

**Review by xabbuh — APPROVED** [review]

**Pull request merged**